code/modules/clothing/suits/hazard_vest.dm

```diff
@@ -0,0 +1,69 @@
+	desc = "A high-visibility vest used in work zones. This one has engineering colors."
+	icon_state = "hazard_engi"
+	item_state = "hazard_engi"
+	allowed = list (/obj/item/flashlight, /obj/item/t_scanner, /obj/item/tank/internals/emergency_oxygen, /obj/item/clipboard, /obj/item/analyzer, /obj/item/screwdriver, /obj/item/radio, /obj/item/wrench, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/wirecutters, /obj/item/rcd, /obj/item/rpd)
+
+/obj/item/clothing/suit/storage/hazardvest/atmos
+	name = "atmospheric hazard vest"
+	desc = "A high-visibility vest used in work zones. This one has atmospheric colors."
+	icon_state = "hazard_atmos"
+	item_state = "hazard_atmos"
+	allowed = list (/obj/item/flashlight, /obj/item/t_scanner, /obj/item/tank/internals/emergency_oxygen, /obj/item/clipboard, /obj/item/analyzer, /obj/item/screwdriver, /obj/item/radio, /obj/item/wrench, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/wirecutters, /obj/item/rcd, /obj/item/rpd)
+
+/obj/item/clothing/suit/storage/hazardvest/ce
+	name = "chief engineer's hazard vest"
+	desc = "A high-visibility vest used in work zones. This one belongs to chief engineer. It's made out of strong material."
+	icon_state = "hazard_CE"
+	item_state = "hazard_CE"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, RAD = 5, FIRE = 50, ACID = 50) // CE is special enough that they can probably get some minor armor
+	allowed = list (/obj/item/flashlight, /obj/item/t_scanner, /obj/item/tank/internals/emergency_oxygen, /obj/item/clipboard, /obj/item/analyzer, /obj/item/screwdriver, /obj/item/radio, /obj/item/wrench, /obj/item/weldingtool, /obj/item/crowbar, /obj/item/wirecutters, /obj/item/rcd, /obj/item/rpd, /obj/item/melee/classic_baton/telescopic) // Same as regular, but also can hold their telebaton
+
+/*CARGO*/
+/obj/item/clothing/suit/storage/hazardvest/cargo
+	name = "cargo hazard vest"
+	desc = "A high-visibility vest used in work zones. This one has cargo colors."
+	icon_state = "hazard_cargo"
+	item_state = "hazard_cargo"
+	allowed = list (/obj/item/flashlight, /obj/item/paper, /obj/item/clipboard, /obj/item/tank/internals/emergency_oxygen, /obj/item/rcs, /obj/item/destTagger, /obj/item/analyzer, /obj/item/wrench, /obj/item/radio)
+
+/obj/item/clothing/suit/storage/hazardvest/qm
+	name = "quartermaster's hazard vest"
+	desc = "A high-visibility vest used in work zones. This one belongs to quartermaster."
+	icon_state = "hazard_QM"
+	item_state = "hazard_QM"
+	allowed = list (/obj/item/flashlight, /obj/item/paper, /obj/item/clipboard, /obj/item/tank/internals/emergency_oxygen, /obj/item/rcs, /obj/item/destTagger, /obj/item/analyzer, /obj/item/wrench, /obj/item/radio, /obj/item/melee/classic_baton/telescopic) // Same as regular, but also can hold their telebaton
+
+/*SECURITY*/
+/obj/item/clothing/suit/storage/hazardvest/sec
+	name = "security hazard vest"
+	desc = "A high-visibility vest used in work zones. This one has security colors. It does not look durable."
```

Comment:
well.. its more durable than the others :D

Reply:
it's less durable than the armor. I want to make it clear.